lxml-stubs/etree.pyi

```diff
@@ -531,13 +531,18 @@ class ParseError(LxmlSyntaxError):
     position: Tuple[int, int]
 
 class XMLSyntaxError(ParseError): ...
-class _Validator: ...
+
+class _Validator:
+    def assert_(self, etree: Union[_Element, _ElementTree]) -> None: ...
```

Comment:
`Union[_Element, _ElementTree]` - this seems worth a dedicated typedef by now, e.g. `_ElementOrTree`. There are lots of places in the API that accept either of the two.

Reply:
Yes, I agree. Done with 31c0159